models/claim_review.rb

```diff
@@ -50,7 +50,14 @@ def self.service_query(service)
   end
 
   def self.delete_by_service(service)
-    ClaimReview.client.delete_by_query(self.service_query(service))["deleted"]
+    retry_count = 0
```

Comment:
I suggest to use a retriable module, such as https://github.com/kamui/retriable

Reply:
Lol, I had no idea that someone built this - great little tool